### DISCUSSION TITLE
Receiving 500 error after POST to /livewire/update

### DISCUSSION THREAD

**Discussion by nickbasham** [livewire/livewire#7453] [1. Help]
Hi everyone, I'm hoping this is a simple issue and just some misconfiguration on my end, but any ideas to point me in the right direction would be greatly appreciated.

I'm implementing Livewire for the first time (v3), replacing several components with slow-loading queries on a dashboard page with lazy-loading Livewire components. Everything is working great in my local dev environment, but I'm running into a 500 error when running in my dev environment on my server.

Livewire is loading correctly and all my components are displaying their placeholder Blade templates as expected, but as soon as Livewire attempts the AJAX POST request to /livewire/update, it returns the 500 error and I get a big black empty box overlaying the page content. The box has classes indicating it's for Livewire errors, but no content is shown.

I've tried every clear command with Artisan and `composer dump-autoload`, but still seeing the same behavior.

Any ideas?

**Comment by JTibblesSoftcon** [comment]
Following laravel 10 and livewire 3 upgrades I'm also getting 500 errors regarding livewire/update when calling a method via wire:click . 
I've added the discussion here: https://github.com/livewire/livewire/discussions/7470
...do you get similar errors? Have you had any luck solving it yet.

**Reply by nickbasham** [reply]
It ended up just being an out of memory error, so not Livewire-related at all.

**Comment by edalzell** [comment]
What's the error in your log?

**Reply by nickbasham** [reply]
I don't know why I didn't think to check my logs…it ended up being a PHP out of memory error. I figured it was something dumb I overlooked, thanks for the suggestion!
